src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -115,11 +115,12 @@
                     (aset xhr (name k) v))
                   xhr))
           call-on-done (fn []
-                         (on-done (let [status-code (.-status xhr)]
+                         (on-done (let [status-code (.-status xhr) raw-response (.-responseText xhr)]
                                     {:xhr xhr
                                      :status-code status-code
                                      :success? (<= 200 status-code 299)
                                      :status-text (.-statusText xhr)
+                                     :raw-response raw-response
```

Comment:
Style nit: either bump `raw-response` down to its own line in the `let`, or just do `:raw-response (.-responseText xhr)`